arrange_schedule.py

```diff
@@ -251,11 +247,9 @@ def find_text_acticity(json_obj):
         #reshape deal result
         return_msg["ans_list"] = deal_result
         
-        db.close()  
         return_msg["result"] = "success"
         return return_msg
     except DB_Exception as e:
```

Comment:
I just wonder that if we move all the sql string and operation to DAO, `except DB_Exception as e` will have any effect?